src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partnerGender = isMarried ? (`${partnerName} is my ${sex === 'm' ? ('wife') : ('husband')}`) : (`I am not married`);
```

Comment:
Would be better to leave only this logic in this variable and use the conditional rendering in JSX 
```suggestion
  const partnerGender = sex === 'm' ? 'wife' : 'husband';
```